Orange/widgets/visualize/owmosaic.py

```diff
@@ -281,8 +281,11 @@ class OWMosaicDisplay(OWWidget):
     variable2 = ContextSetting("", exclude_metas=False)
     variable3 = ContextSetting("", exclude_metas=False)
     variable4 = ContextSetting("", exclude_metas=False)
+    variable_color = ContextSetting("", exclude_metas=False)
     selection = ContextSetting(set())
 
+    color_var = None
```

Comment:
... if this line is needed at all. I may be stupid, by I don't see what `color_var` is doing and whether it's a string or a list of strings.